doc/src/compute_com_chunk.rst

```diff
@@ -8,54 +8,73 @@ Syntax
 
 .. code-block:: LAMMPS
 
-   compute ID group-ID com/chunk chunkID
+   compute ID group-ID com/chunk chunkID keyword args ...
 
 * ID, group-ID are documented in :doc:`compute <compute>` command
 * com/chunk = style name of this compute command
 * chunkID = ID of :doc:`compute chunk/atom <compute_chunk_atom>` command
+* zero or more keyword/value pairs may be appended
+* keyword = *wrap*
+
+  .. parsed-literal::
+
+       *wrap* logical = wrap center of mass back to cell if argument is *yes*, *true*, *on* or *1*, otherwise do not (default)
```

Comment:
should we just choose one? perhaps on some more general docs page, we can state that other arguments can typically be substituted for 'yes'